git/git-batch-merge.yaml

```diff
@@ -0,0 +1,117 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: git-batch-merge
+spec:
+  workspaces:
+    - name: output
+      description: The git repo will be cloned onto the volume backing this workspace
+  params:
+    - name: url
+      description: git url to clone
+      type: string
+    - name: revision
+      description: base git revision to checkout (branch, tag, sha, ref…)
+      default: master
```

Comment:
suggest adding `type: string` in all the params where this is missing

Reply:
Done